tests/realm.cpp

```diff
@@ -493,20 +489,20 @@ TEST_CASE("SharedRealm: schema updating from external changes") {
         }},
     };
 
-    #if 0
     SECTION("newly added columns update table columns but are not added to properties") {
+        // Does this test add any value when column keys are stable?
```

Comment:
Testing that new columns aren't added to `persisted_properties` in non-dynamic schema modes is still useful. That might be sufficiently covered elsewhere?